.github/workflows/examples-providers.yml

```diff
@@ -88,15 +77,19 @@ jobs:
       - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683
 
       # Download wash binary & install to path
-      - uses: actions/download-artifact@fa0a91b85d4f404e444e00e005971372dc801d16
+      - name: install wash (${{ matrix.wash-version }})
+        if: ${{ matrix.wash-version && matrix.wash-version != 'current' }}
+        uses: taiki-e/install-action@acd25891978b4cdaebd139d3efef606d26513b14 # v2.47.0
         with:
-          name: wash-${{ matrix.wash-version }}
-          path: artifacts
-      - name: install wash binary to PATH # Some build tools expect wash to be on the PATH
-        shell: bash
-        run: |
-          chmod +x ./artifacts/wash;
-          echo "$(realpath ./artifacts)" >> "$GITHUB_PATH";
+          tool: wash@${{ matrix.wash-version }}
+
+      - name: install wash (current)
+        if: ${{ matrix.wash-version && matrix.wash-version == 'current' }}
+        uses: taiki-e/cache-cargo-install-action@caa6f48d18d42462f9c30df89e2b4f71a42b7c2c # v2.0.1
+        with:
+          tool: wash-cli
+          git: https://github.com/${{ github.repository }}
```

Comment:
You could use [`${{ github.repositoryUrl }}`](https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/accessing-contextual-information-about-workflow-runs#example-contents-of-the-github-context) for this